lib/helpers.coffee

```diff
@@ -32,9 +32,8 @@ module.exports = Helpers =
           resolve(data.stderr.join(''))
       if isNodeExecutable
         options.env ?= {}
-        for prop in process.env
-          if process.env.hasOwnProperty(prop) and prop isnt 'OS'
-            options.env[prop] = process.env[prop]
+        for prop,value of process.env
```

Comment:
Thanks I missed the change from `for ... in` to `for ... of` because it was hidden inside all the formatting changes.